src/validation.h

```diff
@@ -672,7 +667,7 @@ class CChainState {
     bool ActivateBestChain(
         BlockValidationState& state,
         const CChainParams& chainparams,
-        std::shared_ptr<const CBlock> pblock) LOCKS_EXCLUDED(cs_main);
+        std::shared_ptr<const CBlock> pblock = std::shared_ptr<const CBlock>()) LOCKS_EXCLUDED(cs_main);
```

Comment:
In commit "validation: Remove global ::ActivateBestChain" (7e15a2c88e49ec6719997e2cdd8481ddd4e7f6be)

Would seem clearer to say `= nullptr` if that works and you don't prefer this longer version

Reply:
Fixed!